cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -536,11 +536,19 @@ <h4 class="next-steps-heading">Save for a larger down payment.</h4>
                             block__border-top
                             block__flush-top
                             block__padded-top
-                            note-ctr">
-                    <div class="note">
-                        <p class="alert"><strong>Check your credit report.</strong></p>
-                        If you haven’t checked your credit report recently, <a id="annualcreditreport" class="external-link" href="https://annualcreditreport.com" target="_blank" rel="noopener noreferrer">do so now</a>. If you find errors, <a id="ask-dispute-credit-report" href="/askcfpb/314/how-do-i-dispute-an-error-on-my-credit-report.html" target="_blank" rel="noopener noreferrer">get them corrected</a> <strong>before</strong> you apply for a mortgage.
-                    </div> <!-- /.note -->
+                            credit-note-ctr">
+                    <div class="credit-note
+                                m-notification m-notification__warning
+                                m-notification__visible"
+                        role="alert">
+                        <span class="m-notification_icon cf-icon"></span>
+                        <div class="m-notification_content">
+                            <div class="h4 m-notification_message">Check your credit</div>
+                            <div class="h4 m-notification_explanation">
+                                If you haven’t checked your credit report recently, <a id="annualcreditreport" class="external-link" href="https://annualcreditreport.com" target="_blank" rel="noopener noreferrer">do so now</a>. If you find errors, <a id="ask-dispute-credit-report" href="/askcfpb/314/how-do-i-dispute-an-error-on-my-credit-report.html" target="_blank" rel="noopener noreferrer">get them corrected</a> before you apply for a mortgage.
```

Comment:
Trivial. It would be helpful to wrap this long line.

Reply:
Is `id="annualcreditreport"` used?

Reply:
Also `id="ask-dispute-credit-report"`? Are they there as fragment identifiers in the URL for sharing or something?

Reply:
Removed ids in https://github.com/cfpb/cfgov-refresh/pull/4100/commits/70e143a4407fbde245f74d2c9023ab588016c378.